### PR TITLE
Add `dbtool` documentation

### PR DESCRIPTION
**Pull request by elsirion**
People keep asking me how to use it, I take this as a sign that I should document it better :laughing:

### CODE REVIEW

File: docs/debugging.md [hunk 1]
```diff
@@ -5,3 +5,5 @@
 You can use any other regex inplace of `ContractKey.*`, it matches with debug format of key.
 
 - To log all api requests that server `RUST_LOG="fedimint_server::request"`
+
+- [Inspect and manipulate the database using `dbtool`](../fedimint-dbtool/README.md) (low level, see `fedimint-dbdump` for a higher-level inspection tool)
```
Comment (m1sterc001guy) [inline docs/debugging.md:9]: Haha I actually added some documentation in my PR too :)

FYI in that PR I actually merged dbtool and dbump - feel free to keep here though and I can resolve merge conflicts

https://github.com/fedimint/fedimint/pull/1523

File: fedimint-dbtool/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+**CAUTION: `dbtool` is a potentially destructive expert-tool, handle with care and if in doubt make a copy of your
+database.**
+
+For database layout see [`database.md`](../docs/database.md), but verify the information in the code before taking
+potentially destructive action.
+
+## Usage
+```
+$ dbtool --help
+Tool to inspect and manipulate rocksdb databases. All binary arguments (keys, values) have to be hex encoded
```
Comment (m1sterc001guy) [inline fedimint-dbtool/README.md:12]: Do you have a command for taking some data you want to write and hex encoding it?

File: fedimint-dbtool/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+## Deleting multiple elements
+
+Other than the `list` command, the `delete` command only works on single keys. To delete entire key prefixes you can use
+standard unix tools to build that functionality:
+
+```bash
+dbtool <DATABASE> list <PREFIX> | cut -d ' ' -f 1 | xargs -n 1 -- dbtool <DATABASE> delete
+```
+
+* `cut` selects a colum from the space-separated output of `dbtool`
```
Comment (m1sterc001guy) [inline fedimint-dbtool/README.md:38]: typo on column

**Review by m1sterc001guy — APPROVED** [review]

File: fedimint-dbtool/README.md (hunk 2 above)
Comment (elsirion) [reply fedimint-dbtool/README.md:12]:
```bash
$ echo -n "my binary data ..." | xxd -ps -c 0
6d792062696e6172792064617461202e2e2e
```

But tbh, whenever I was writing I had to manually construct these hex strings anyway by encoding the right integers etc.

File: fedimint-dbtool/README.md (hunk 3 above)
Comment (elsirion) [reply fedimint-dbtool/README.md]: fixed

File: fedimint-dbtool/README.md (hunk 2 above)
Comment (elsirion) [reply fedimint-dbtool/README.md:12]: added a section on it

**Comment by elsirion** [comment]
I think it's still useful, just a question of how annoying rebasing would be. Feel free to just copy the useful parts into your PR and close this one if it's significantly easier. Otherwise we can merge this and you rebase, so we have the docs a bit earlier.

**Review by m1sterc001guy — APPROVED** [review]

**Comment by justinmoon** [comment]
dbtool saved my tail today 😂

**Pull request merged**